server/queues/tasks/CollectionSubscriptionTask.ts

```diff
@@ -0,0 +1,52 @@
+import { Transaction } from "sequelize";
+import { SubscriptionType } from "@shared/types";
+import { createContext } from "@server/context";
+import Logger from "@server/logging/Logger";
+import { Collection, Subscription, User } from "@server/models";
+import { can } from "@server/policies";
+import { sequelize } from "@server/storage/database";
+import { CollectionUserEvent } from "@server/types";
+import BaseTask from "./BaseTask";
+
+export default class CollectionSubscriptionTask extends BaseTask<CollectionUserEvent> {
```

Comment:
Lets rename these tasks to better reflect their function – e.g. `CollectionSubscriptionRemoveUserTask`

We can also remove the `event.name` check as we're in charge of when this runs.